AutoSteamApp/Helpers/StaticHelpers.cs

```diff
@@ -0,0 +1,179 @@
+        public static void SetConsoleTitle()
+        {
+            Version ver = ConfigurationReader.ApplicationVersion;
+            string title = "Auto Steamworks Version:" + ver.Major + "." + ver.Minor;
+            title += "        ";
+            title += "Supported MHW:IB Version: " + MHWMemoryValues.SupportedGameVersion;
+            Console.Title = title;
+        }
+
+        /// <summary>
+        /// Converts the byte sequence to an array of Virtual Key Codes using the index of the key as the order in which to press
+        /// <param name="sequence"></param>
+        /// <returns></returns>
+        public static VirtualKeyCode[] KeyCodeSequenceFromBytes(byte[] sequence)
+        {
+
+            Dictionary<int, VirtualKeyCode> dict = new Dictionary<int, VirtualKeyCode>();
+
+            if (ConfigurationReader.IsAzerty)
+            {
+                dict.Add(sequence[0], VirtualKeyCode.VK_Q);
+                dict.Add(sequence[1], VirtualKeyCode.VK_Z);
+                dict.Add(sequence[2], VirtualKeyCode.VK_D);
+            }
+            else
+            {
+                dict.Add(sequence[0], VirtualKeyCode.VK_A);
+                dict.Add(sequence[1], VirtualKeyCode.VK_W);
+                dict.Add(sequence[2], VirtualKeyCode.VK_D);
+            }
+
+            VirtualKeyCode[] retVal = dict.OrderBy(x => x.Key).Select(y => y.Value).ToArray();
+            // Return the virtual key code values ordered by the index assigned from the byte sequence
+            Log.Debug("Sequence found: [" + string.Join(", ", retVal.Select(x => x.ToString())) + "]");
+            return retVal;
+        }
+
+        /// <summary>
+        /// Generates a random sequence of three key codes
+        /// </summary>
+        /// <returns></returns>
+        public static VirtualKeyCode[] RandomSequence()
+        {
+            VirtualKeyCode[] retVal = new VirtualKeyCode[3];
+            if (ConfigurationReader.IsAzerty)
+            {
+                retVal[0] = VirtualKeyCode.VK_Q;
+                retVal[1] = VirtualKeyCode.VK_Z;
+                retVal[2] = VirtualKeyCode.VK_D;
+            }
+            else
+            {
+                retVal[0] = VirtualKeyCode.VK_A;
+                retVal[1] = VirtualKeyCode.VK_W;
+                retVal[2] = VirtualKeyCode.VK_D;
+            }
+            return retVal.Shuffle();
+        }
+
+        /// <summary>
+        /// Simulates a key down, followed by key up event for a key
+        /// </summary>
+        /// <param name="sim">The input simulator to use.</param>
+        /// <param name="key">The key to press.</param>
+        /// <param name="delay">The delay to wait between key down and key up.</param>
+        public static void PressKey(InputSimulator sim, VirtualKeyCode key, int delay = 0)
+        {
+            // Im not sure if thread.sleep eats up allocation etc. resources when called with 0 as an argument
+            // So I've decided to skip it if the delay is 0
```

Comment:
As you would expect, there is a difference between KeyDown + KeyUp vs KeyPress - you can see the exact implementation here https://github.com/GregsStack/InputSimulatorStandard/blob/master/src/GregsStack.InputSimulatorStandard/KeyboardSimulator.cs - now, What i noticed is that using KeyDown and KeyUp with a delay (even if delay is 0) will minimize the input lag - thus making the game accept the key pressing faster than using KeyPress - This is probably generated by the way the game interprets the button pressing - (see this for more info : https://github.com/AdiBorsos/AutoSteamworks/issues/12#issuecomment-600473052 )